Sources/StreamChatUI/UIConfig+MessageComposer.swift

```diff
@@ -10,8 +10,8 @@ public extension _UIConfig {
             _ChatMessageComposerVC<ExtraData>.self
         public var messageComposerView: _ChatMessageComposerView<ExtraData>.Type =
             _ChatMessageComposerView<ExtraData>.self
-        public var messageInputView: _ChatMessageComposerInputContainerView<ExtraData>
-            .Type = _ChatMessageComposerInputContainerView<ExtraData>.self
+        public var messageInputView: _ChatInputContainerView<ExtraData>
+            .Type = _ChatInputContainerView<ExtraData>.self
```

Comment:
I think we need to move this to the root of the UIConfig. Only `messageComposerView` should live here

Reply:
@nuno-vieira Ideally nothing lives here and the `UIConfig` is just one level deep.

Reply:
Yes exactly, we should aim to remove all `UIConfig+X` sections. The `UIConfig` should just act like a design system, where each component can be used and composed in any part of the app 👍

Reply:
I aim to refactor UIConfig in a follow up PR, wdyt?